packages/extension-ui/src/components/Button.tsx

```diff
@@ -37,7 +37,10 @@ function Button({
       return;
     }
 
-    onClick && onClick();
+    // TODO: check why is it giving errors
```

Comment:
What kind of errors?

Reply:
![image](https://user-images.githubusercontent.com/50427204/222080135-19adea87-2961-4985-b8ee-ffa8dd52f410.png)

Reply:
I see. Judging by [that](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-floating-promises.md#ignorevoid), I suppose adding the `void` operator should silence the error:
```ts
void onClick()...
```
TIL. Quite nice way of avoiding _forgetting_ about awaiting 👍